src/test/scala/chiselTests/aop/SelectSpec.scala

```diff
@@ -139,5 +141,17 @@ class SelectSpec extends ChiselFlatSpec {
     )
   }
 
+  "Blackboxes" should "be supported in Select.instances" in {
+    class BB extends ExtModule { }
+    class Top extends RawModule {
+      val bb = Module(new BB)
+    }
+    val top = ChiselGeneratorAnnotation(() => {
+      new Top()
+    }).elaborate(1).asInstanceOf[DesignAnnotation[Top]].design
```

Comment:
This `elaborate(1)` threw me for a loop... Maybe `elaborate.head`? (That's a nit.)

Reply:
Its not head, its the second in the list.. `elaborate.tail.head`?

Reply:
Ugh... not thinking (or thinking in tuples). `.tail.head` isn't amazing either. Pattern match (`case _ :: a :: _ => `) might be the most readable. Not worth it, though. I'd just merge it.